src/language_server.rs.in

```diff
@@ -0,0 +1,1135 @@
+
+use std::collections::HashMap;
+use serde;
+use serde::de::Error;
+use serde_json::Value;
+
+#[derive(Deserialize)]
+pub struct CancelParams {
+    /**
+     * The request id to cancel.
+     */
+    pub id: String,
+}
+
+#[derive(Deserialize)]
+pub struct DidChangeTextDocumentParams {
+    /**
+     * The document that did change. The version number points
+     * to the version after all provided content changes have
+     * been applied.
+     */
```

Comment:
Might want to run rustfmt on this, copying over our setting from `gluon-lang/gluon`